api/src/main/java/ca/bc/gov/educ/api/ruleengine/config/GradRuleEngineApiConfig.java

```diff
@@ -1,28 +1,52 @@
 package ca.bc.gov.educ.api.ruleengine.config;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
+import ca.bc.gov.educ.api.ruleengine.util.LogHelper;
+import ca.bc.gov.educ.api.ruleengine.util.RuleEngineApiConstants;
 import org.modelmapper.ModelMapper;
 import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
 import org.springframework.web.client.RestTemplate;
-
-import java.util.TimeZone;
+import org.springframework.web.reactive.function.client.ExchangeFilterFunction;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.netty.http.client.HttpClient;
 
 @Configuration
 public class GradRuleEngineApiConfig {
 
+    RuleEngineApiConstants constants;
+    LogHelper logHelper;
+
     @Bean
     public ModelMapper modelMapper() {
         return new ModelMapper();
     }
 
+    @Bean
+    public WebClient webClient() {
```

Comment:
Don't need a webclient or log on this as there are no callouts. You can revert changes to this class.